src/components/ExportButton.js

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import { Button } from 'semantic-ui-react';
+
+function ExportButton({ exportFunction }) {
```

Comment:
How about importing the `exportRecommendations` function directly in this file instead of passing it as a prop.

Reply:
I actually deliberately inject it as a prop because we will have another export button for the description, so there we can pass in a exportDescription function

Reply:
Sounds reasonable :)